lib/drawNametag.ts

```diff
@@ -0,0 +1,70 @@
+// drawNametag.ts
+
+export default function drawNametag(nameTagStatus:boolean, currentNameTag:string[], selectedWaveHand: number | null, waveHands: string[]): ImageData {
```

Comment:
What do you think about an API like this instead:
```suggestion
export interface NameTagBadge {
  visible: boolean;
  fullName?: string;
  preferredName?: string;
  pronouns?: string;
  disclosure?: string;
}

export interface HandWaveBadge {
  visible: boolean;
  text: string;
}
export default function drawBadges(nametag: NameTagBadge, handWave: HandWaveBadge): ImageData {
```